homework-solutions/js/hw-4/task-1.js

```diff
@@ -1,5 +1,12 @@
 /*
 
- - Создайте переменную salary со значением 1000
-  - Создайте переменную grade, которая должна получить значение "middle" если salary больше или равна 1000, и значение "junior" - если меньше
+- Создайте переменную salary со значением 1000
+- Создайте переменную grade, 
+которая должна получить значение "middle", если salary больше или равна 1000, 
+и значение "junior" - если меньше
   */
+  const salary = 1000;
+  if (salary >= 1000)
+{console.log(grade = "middle");}
```

Comment:
Во-первых поставь пожалуйста расширение Prettier в вскоде и проходись им по коду)

```suggestion
{
  grade = "middle";
  console.log(grade);
}
```